Cargo.toml

```diff
@@ -113,3 +113,64 @@ members = [
 	"orml/vesting",
 ]
 exclude = [ "runtime/wasm" ]
+
+# The list of dependencies below (which can be both direct and indirect dependencies) are crates
+# that are suspected to be CPU-intensive, and that are unlikely to require debugging (as some of
+# their debug info might be missing) or to require to be frequently recompiled. We compile these
+# dependencies with `opt-level=3` even in "dev" mode in order to make "dev" mode more usable.
+# The majority of these crates are cryptographic libraries.
+#
+# If you see an error mentioning "profile package spec ... did not match any packages", it
+# probably concerns this list.
+#
+# This list is ordered alphabetically.
```

Comment:
add a link to source file so we know where is this come form and easier to keep in sync

Reply:
Source file link added.